src/ziggurat/server/middleware.clj

```diff
@@ -40,3 +42,11 @@
           response-status   (:status response)]
       (metrics/increment-count ["http-server" "requests-served"] "count" {:request-uri request-uri :response-status (str response-status)})
       response)))
+
+(defn- swagger-enabled? []
+  (get-in-config [:http-server :middlewares :swagger :enabled]))
```

Comment:
(true? (get-in-config [:http-server :middlewares :swagger :enabled]))) it will force the config to have a value of true only

Reply:
Done